docs/BigQueryDropDataset-action.md

```diff
@@ -0,0 +1,34 @@
+# BigQuery Drop Dataset
+
+#### **Description**
+
+Drop a BigQuery dataset.
+
+#### **Basic**
+
+Following are properties used to configure this plugin
```

Comment:
remove or change to:

The following properties can be used to configure this plugin: